mantidimaging/gui/windows/main/view.py

```diff
@@ -331,6 +331,7 @@ def load_shuttercounts_dialog(self):
         QMessageBox.information(
             self, "Load complete", f"{selected_file} was loaded as a shutter count file into "
             f"{stack_to_add_shuttercounts_to}.")
+        self.spectrum_viewer.handle_shuttercount_change()
```

Comment:
This fails if the spectrum viewer has not been opened. It ought to emit a signal here, and the spectrum viewer should hook that up to `handle_shuttercount_change`

Reply:
Just added an if statement to check if self.spectrum exists which matches the style lower down in the view.